SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -110,7 +110,7 @@ private static PlatformRefreshTokenCredential GetPlatformRefreshTokenCredential(
 
         private static int CreateDeployment(string[] args, bool useChinaPlatform)
         {
-            bool launchSimPlayerDeployment = args.Length == 15;
+            bool launchSimPlayerDeployment = args.Length == 16 || args.Length == 15;
```

Comment:
Might be worth adding a comment that the args differ based on optional args?

Reply:
Good one - adding it now